mantidproject/mantid
# New "energy bands" tab in tomography GUI

Extends the "energy bands" tab of the tomo GUI and uses ImggAggWavelengths.
- [x] #15873 is required for this and should be merged in first.

**To test:**
- Tests should pass, including the new unit test cases
- Go to Main Menu ->Interfaces->Diffraction->Tomographic reconstruction->"Energy bands" tab
- Select an input path using the same example/test data as in PR #15873. Select as ouput path any existing directory where it is ok to write the files. Select for example "uniform bands", with a value of 2.
- Start the aggregation process.
- When it finishes it will show a pop up to notify the user that the
  results are ready.
- If you give empty or wrong paths it should give an informative error message.
- The "TOF boundaries" options is disabled for now - we need headers in the input files which are not available at the moment.
- The input and output formats are currently forced to "FITS".
- The interface should feel usable, and it should remember all settings.
- Check documentation updates

Fixes #13467.

Release notes: https://github.com/mantidproject/mantid/commit/20c7f46aa0829edafb6241349132180bc1984e34

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
